go.mod

```diff
@@ -40,10 +40,8 @@ require (
 	github.com/mysteriumnetwork/go-wondershaper v1.0.1
 	github.com/mysteriumnetwork/metrics v0.0.3
 	github.com/mysteriumnetwork/payments v0.0.13
-	github.com/nats-io/gnatsd v1.4.1 // indirect
-	github.com/nats-io/go-nats v1.4.0
-	github.com/nats-io/nats-server v1.4.1
-	github.com/nats-io/nuid v1.0.1-0.20180712044959-3024a71c3cbe // indirect
+	github.com/nats-io/nats-server/v2 v2.1.7
+	github.com/nats-io/nats.go v1.10.0
```

Comment:
Is our NATS server compatible with this new nats client?

Reply:
I did a check to establish connection and it worked fine. E2E tests should cover this too.